modules/java/generator/src/cpp/Mat.cpp

```diff
@@ -2157,7 +2723,6 @@ JNIEXPORT jstring JNICALL Java_org_opencv_core_Mat_nDump
 {
     static const char method_name[] = "Mat::nDump()";
     try {
-        LOGD("%s", method_name);
```

Comment:
Why removed?

Reply:
Made some changes to this method then reverted them - this line got lost along the way.
Will put it back.